src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -116,6 +116,7 @@ public OutsideRuntimeClient(ClientConfiguration cfg, bool secondary = false)
             services.AddFromExisting<IInternalGrainFactory, GrainFactory>();
             services.AddFromExisting<IGrainReferenceConverter, GrainFactory>();
             services.AddSingleton<MessageFactory>();
+            services.AddSingleton<Func<string, Logger>>(LogManager.GetLogger);
```

Comment:
This method was only available to internals, do we want to open it up to the public? I'm fine with it, just highlighting this change